src/App.js

```diff
@@ -1,23 +1,64 @@
-import logo from './logo.svg';
 import './App.css';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import Image from './Image';
+import Form from './Form';
+import title from './title.png';
 
 function App() {
+
+  const [pokemon, setPokemon] = useState({
+    name: "",
+    img: "",
+  });
+
+  const [num, setNum] = useState(123);
+
+  const [userChoice, setUserChoice] = useState('');
+
+  const [counter, setCounter] = useState(0);
+
+  const randomNumber = (num) => {
+    return Math.floor(Math.random() * num);
+  }
+
+  useEffect (() => {
+    setNum(randomNumber(252));
+  }, []);
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    setNum(randomNumber(252));
+    setUserChoice('');
+    if (pokemon.name === userChoice) {
```

Comment:
maybe wrap this in another conditional to check if user wrote a name. 
if(userchoice ===""){ do your code}else{alert("write a name"}